code/modules/species/protean/protean_blob.dm

```diff
@@ -145,16 +145,10 @@
 	return FALSE //ok so tasers hurt protean blobs what the fuck
 
 /mob/living/simple_mob/protean_blob/adjustBruteLoss(var/amount,var/include_robo)
-	if(humanform)
-		humanform.adjustBruteLossByPart(amount, BP_TORSO)
-	else
-		..()
+	return humanform? humanform.take_targeted_damage(brute = amount, body_zone = BP_TORSO) : ..()
```

Comment:
Making it less readable why?

Reply:
it should be returning a value, it wasn't doing that before